docs/general/nft.md

```diff
@@ -0,0 +1,18 @@
+---
+sidebar_position: 5
+sidebar_label: 'NFTs'
```

Comment:
The same here. Maybe `NFT` is better than `NFTs`.

Reply:
ok, updated